### PR TITLE
Change github workflow

### PR DESCRIPTION
**Pull request by richardm-stripe**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,6 +11,10 @@ test: vendor
 	vendor/bin/phpunit
 .PHONY: test
 
+ci-test: vendor
+	php --version
```
Comment (pakrym-stripe) [inline Makefile:15]:
```suggestion
```

File: tests/Stripe/GeneratedExamplesTest.php [hunk 2]
```diff
@@ -4010,4 +4010,16 @@ public function testUpdateWebhookEndpoint()
...
+    {
+        $this->expectsRequest(
+            'post',
+            '/v1/tax/transactions/create_from_calculation'
+        );
+        $result = $this->client->tax->transactions->createFromCalculation(
+            ['calculation' => 'xxx', 'reference' => 'yyy']
+        );
+        static::assertInstanceOf(\Stripe\Tax\Transaction::class, $result);
+    }
```
Comment (pakrym-stripe) [inline tests/Stripe/GeneratedExamplesTest.php:4024]:
```suggestion
```

File: Makefile [hunk 3]
```diff
@@ -11,6 +11,9 @@ test: vendor
 	vendor/bin/phpunit
 .PHONY: test
 
+ci-test: vendor
+	./build.php $$AUTOLOAD
+
```
Comment (pakrym-stripe) [inline Makefile:16]:
```suggestion
	./build.php $$AUTOLOAD
.PHONY: ci-test

```

**Review by pakrym-stripe — APPROVED** [review]

**Review by pakrym-stripe — CHANGES_REQUESTED** [review]
Github complains about invalid yaml

**Review by anniel-stripe — APPROVED** [review]

**Pull request merged**